src/foam/u2/dialog/ActionReference.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.dialog',
```

Comment:
Maybe ‘foam.u2’ is more appropriate as a package

Reply:
Right, 'cause this could be useful anywhere - updated